Capture editor.action.triggerSuggest command and make it no-op

### Server Version

```
08dbe84 (unreleased)
```


### Client Version

```
Sublime Text (LSP plugin) v1.1.0
```

### Terraform Configuration Files

```hcl
provi<Enter>
```

### Log Output

```
terraform: 2020/11/18 09:00:19 server.go:174: Processing 1 requests
terraform: 2020/11/18 09:00:19 server.go:265: Checking request for "workspace/executeCommand": {"command":"editor.action.triggerSuggest"}
terraform: 2020/11/18 09:00:19 rpc_logger.go:29: Incoming request for "workspace/executeCommand" (ID 3): {"command":"editor.action.triggerSuggest"}
terraform: 2020/11/18 09:00:19 rpc_logger.go:45: Error for "workspace/executeCommand" (ID 3): [-32601] method not found: command handler not found for "editor.action.triggerSuggest"
```

### Expected Behavior

No-op in editors/clients which don't register `editor.action.triggerSuggest` themselves.

### Actual Behavior

Error popup.

![Screenshot 2020-11-18 at 08 44 50](https://user-images.githubusercontent.com/287584/99508526-99537880-297c-11eb-8fb4-2fadfc3bf838.png)

--- 

This has emerged as part of the latest ST LSP plugin, particularly thanks to https://github.com/sublimelsp/LSP/pull/1451

--- a/langserver/handlers/execute_command.go
+++ b/langserver/handlers/execute_command.go
@@ -44,6 +44,13 @@ func (h executeCommandHandlers) Get(name, commandPrefix string) (executeCommandH
 }
 
 func (lh *logHandler) WorkspaceExecuteCommand(ctx context.Context, params lsp.ExecuteCommandParams) (interface{}, error) {
+	if params.Command == "editor.action.triggerSuggest" {
+		// If this was actually received by the server, it means the client
+		// does not support explicit suggest triggering, so we fail silently
+		// TODO: Revisit once https://github.com/microsoft/language-server-protocol/issues/1117 is addressed
+		return nil, nil
+	}
+
 	commandPrefix, _ := lsctx.CommandPrefix(ctx)
 	handler, ok := handlers.Get(params.Command, commandPrefix)
 	if !ok {

